tests/src/runner/podman-desktop-runner.ts

```diff
@@ -77,6 +79,47 @@ export class PodmanDesktopRunner {
     }
   }
 
+  async getApplicationState(): Promise<{
+    isVisible: boolean;
+    isDevToolsOpened: boolean;
+    isCrashed: boolean;
+    isMaximized: boolean;
+  }> {
+    const window: JSHandle<BrowserWindow> = await this.getBrowserWindow();
+
+    return await window.evaluate(
+      (
+        mainWindow,
+      ): Promise<{ isVisible: boolean; isDevToolsOpened: boolean; isCrashed: boolean; isMaximized: boolean }> => {
+        const getState = () => ({
+          isVisible: mainWindow.isVisible(),
+          isDevToolsOpened: mainWindow.webContents.isDevToolsOpened(),
+          isCrashed: mainWindow.webContents.isCrashed(),
+          isMaximized: mainWindow.isMaximized(),
+        });
+
+        return new Promise(resolve => {
+          resolve(getState());
+        });
+      },
+    );
+  }
+
+  async setupApplication(): Promise<void> {
+    const window: JSHandle<BrowserWindow> = await this.getBrowserWindow();
+
+    await window.evaluate((mainWindow): Promise<void> => {
+      mainWindow.webContents.closeDevTools();
+      mainWindow.maximize();
```

Comment:
I would not maximize as the goal is to really check with what is displayed by default to the user

Reply:
np.